plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/topo_diff.py

```diff
@@ -187,6 +310,20 @@ def get_ufm_links(ufm_port):
     else:
         return response.json, SUCCESS_CODE
 
+def upload_topoconfig_file(ufm_port, payload):
+    '''
+    Upload topoconfig file to the UFM server
+    :param ufm_port:
+    '''
+    ufm_host = "127.0.0.1:{}".format(ufm_port)
+    ufm_protocol = "http"
+    resource = "/Topology_Compare/master_topology"
+    headers = {"X-Remote-User": "ufmsystem"}
```

Comment:
to avoid code duplication, you can create a session and use it in both places, smth like
HOST = f"127.0.0.1:{ufm_port}"
PROTOCOL = "http"
SESSION = requests.Session()
SESSION.headers = {"X-Remote-User": "ufmsystem"}

Reply:
next time